doc/release_notes_0.3.rst

```diff
@@ -0,0 +1,677 @@
+.. _release_notes_0.3:
+
+ACRN v0.3 (Nov 2018)
+####################
+
+We are pleased to announce the release of Project ACRN version 0.3.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline
+embedded development through an open source platform. Check out the
+:ref:`introduction` for more information.
+
+
+All project ACRN source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, and documentation.
+You can either download this source code as a zip or tar.gz file (see
+the `ACRN v0.3 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v0.3>`_ or
+use Git clone and checkout commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v0.3
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v0.3 documents can be found at
+https://projectacrn.github.io/0.3/.  Documentation for the latest
+(master) branch is found at https://projectacrn.github.io/latest/.
+
+
+Version 0.3 new features
+************************
+
+:acrn-1179	- RPMB key passing
```

Comment:
These should all be bullet list items using the ``:acrn-issue:`` role, like this:

    - :acrn-issue:`1179` - RPMB key passing
    - :acrn-issue:`1180` - vFastboot release version 0.9